monitoring/alerts/updatealerts.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
+
+if [ -z "$1" ]
+  then
+    echo "No target evironment argument supplied. Please provide target environment (e.g. pre, dev)"
+    exit;
```

Comment:
Optionally might consider returning `-1` or a non-zero value in case of error to be POSIX compliant.